projects/orquestra-workflow-runtime/README.md

```diff
@@ -0,0 +1,32 @@
+# Orquestra Workflow SDK
+
+## What is it?
+
+`orquestra-sdk` is a Python library for expressing and executing computational workflows locally and on the [Orquestra](https://www.zapatacomputing.com/orquestra) platform.
+
+`orquestra-sdk` provides:
+
+- A Python DSL to express your workflows
+- An API for managing and using secrets inside workflows
+- An API for executing and managing your workflows
+- A CLI tool for executing and managing workflows
+
+## Installation
+
+Orquestra Workflow SDK is published to PyPI and should be installed from there via `pip`:
+
+```bash
+pip install "orquestra-sdk[all]"
+```
+
+## Usage
+
+Please refer to the [Orquestra Workflow SDK docs](https://docs.orquestra.io/docs/core/sdk/).
+
+## Bug Reporting
+
+If you'd like to report a bug/issue please create a [new issue using one of the templates](https://github.com/zapata-engineering/orquestra-sdk/issues).
+
+## Contributing
+
+Please see our [CONTRIBUTING.md](CONTRIBUTING.md) for more information on contributing to Orquestra Workflow SDK.
```

Comment:
**nitpick**: this file doesn't seem to describe `orquestra-workflow-runtime`. We don't have to be very verbose about it, but no information is better than misleading information:
```suggestion
# Orquestra Workflow Runtime

Executes Orquestra workflows. This library isn't meant to be used directly by external users, please use [orquestra-sdk](https://github.com/zapata-engineering/orquestra-sdk/tree/main/projects/orquestra-sdk) instead.
```